README.md

```diff
@@ -1,217 +1,17 @@
 # hello-vault-go
 
-This is a sample application that demonstrates how to authenticate to and
-retrieve secrets from HashiCorp [Vault][vault].
+HashiCorp [Vault](https://www.vaultproject.io/) runnable examples in Go.
 
-## Prerequisites
+## [Quick Start](./quick-start/)
 
-1. [`docker`][docker] to easily run the application in the same environment
-   regardless of your local operating system
-1. [`docker compose`][docker-compose] to easily set up all the components of the
-   demo (the application's web server, the Vault server, the database, etc.) all
-   at once
-1. [`curl`][curl] to test our endpoints
-1. [`jq`][jq] _(optional)_ for prettier `JSON` output
+This is a simple example of reading and writing your first secret!
 
-## Try it out
+## [Sample Application](./sample-app/)
 
-> **WARNING**: The Vault server used in this setup is configured to run in
-> `-dev` mode, an insecure setting that allows for easy testing.
+Here you will find a more production-like example that demonstrates many
```

Comment:
I wonder if we should err on the side of saying "realistic" instead of "production-like" just since there's some obvious "not for production" things we've noted here and there in the sample app.

Reply:
Fair point, will update 👍